src/utils/voice_recorder.py

```
@@ -0,0 +1,187 @@
+        Returns:
+            bool: 是否成功連接
+        """
+        if self.is_connected:
+            raise RuntimeError("已經連接到語音頻道")
+
+        try:
+            self.voice_client = voice_client
+            self.is_connected = True
+            self.start_time = time.time()
+
+            logger.info(f"已連接到語音頻道 (Guild: {guild_id})")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)